### ISSUE TITLE
Finder indicator not running on desktop

### ISSUE THREAD

**Issue by parakrammohan** [mydockfinder/mydockfinder#611]
I understand that it the finder app's indicator should be on at all times on dock (even if file explorer is closed) but my finder app's indicator does not show when the file explorer app is closed. How can I fix this?

**Comment by mydockfinder** [comment]
What does that mean, can you take a screenshot or record a video?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月12日(星期二) 下午3:57
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] Finder indicator not running on desktop (Issue #611)





 
I understand that it the finder app's indicator should be on at all times on dock (even if file explorer is closed) but my finder app's indicator does not show when the file explorer app is closed. How can I fix this?

**Comment by parakrammohan** [comment]
![with](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/80622626/b622952b-a8cd-49fe-bd66-cfdb1c50fef3)
with.png ^^
![Without](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/80622626/dd838982-6a25-4d54-a81e-9febf0d6f41f)
without.png ^^

(Indicator is the small dot under the app icon)

English:
As you can see in (without.png) the indicator for file explorer/finder is not on. And the indicator is only on when file explorer is running in (with.png).

Since MyDockFinder acts as a skin for mac, in which the indicator for finder should always be running (even if the finder app itself is not open), I believe the finder app's indicator should be present even in without.png.

However this is not the case, how can I fix this?

Mandarin:
正如您在（without.png）中看到的，文件资源管理器/查找器的指示器未打开。并且该指示灯仅在文件资源管理器运行时亮起（with.png）。

由于 MyDockFinder 充当 mac 的皮肤，其中 finder 的指示器应始终运行（即使 finder 应用程序本身未打开），我相信 finder 应用程序的指示器即使在 without.png 中也应存在。

但事实并非如此，我该如何解决这个问题？

**Comment by mydockfinder** [comment]
I understand, but the normal situation should be to display the indicator all the time, this indicator when there is no window detection standard is whether the system tray has a network, Bluetooth, battery system tray icon
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月13日(星期三) 下午4:03
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Finder indicator not running on desktop (Issue #611)





 
with.png (view on web)
 Without.png (view on web)
 
English:
 As you can see in (without.png) the indicator for file explorer/finder is not on. And the indicator is only on when file explorer is running in (with.png).
 
Since MyDockFinder acts as a skin for mac, in which the indicator for finder should always be running (even if the finder app itself is not open), I believe the finder app's indicator should be present even in without.png.
 
However this is not the case, how can I fix this?
 
Mandarin:
 正如您在（without.png）中看到的，文件资源管理器/查找器的指示器未打开。并且该指示灯仅在文件资源管理器运行时亮起（with.png）。
 
由于 MyDockFinder 充当 mac 的皮肤，其中 finder 的指示器应始终运行（即使 finder 应用程序本身未打开），我相信 finder 应用程序的指示器即使在 without.png 中也应存在。
 
但事实并非如此，我该如何解决这个问题？

**Comment by parakrammohan** [comment]
So what can I do to ensure File explorer/finder's indicator is running at all times please? Can I somehow change this standard? (since my Desktop PC does not have a battery, and I use windows 11 so there is no network option in system tray)

**Comment by mydockfinder** [comment]
There should be network icons.,It's just that in win11, the system is hidden.,Or I'll modify the detection method of the explorer in the next version.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月13日(星期三) 下午4:38
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Finder indicator not running on desktop (Issue #611)





 
So what can I do to ensure File explorer/finder's indicator is running at all times please? Can I somehow change this standard? (since my Desktop PC does not have a battery, and I use windows 11 so there is no network option in system tray)
 
&mdash;

**Comment by parakrammohan** [comment]
ok

**Comment by parakrammohan** [comment]
Hi, have you tried fixing it yet?

**Comment by mydockfinder** [comment]
Yes, the latest version is not fixed, the latest version I added to detect whether the system taskbar exists, if the taskbar exists the explorer icon indicator will not disappear




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月27日(星期三) 晚上6:58
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Finder indicator not running on desktop (Issue #611)





 
Hi, have you tried fixing it yet?
 
&mdash;

**Comment by parakrammohan** [comment]
Thanks

**Comment by parakrammohan** [comment]
![image](https://github.com/mydockfinder/mydockfinder-for-Win10-Win11/assets/80622626/838add2e-e313-4afe-83c9-262bcfcc6b81)
Hi, why is it called explorer? Am I able to change this? (to finder) I cant seem to be able to rename explorer to finder somehow. Pls help. Thanks

**Comment by mydockfinder** [comment]
Because of infringement




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月27日(星期三) 晚上11:37
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] Finder indicator not running on desktop (Issue #611)





 
image.png (view on web)
 Hi, why is it called explorer? Am I able to change this? (to finder) I cant seem to be able to rename explorer to finder somehow. Pls help. Thanks
 
&mdash;

**Comment by parakrammohan** [comment]
Ok is it possible for you to add a rename function? so that you are not infringing it but I can still change it to suit my theme

**Comment by parakrammohan** [comment]
Hi
